gnosis-imap/gnosis-imap-spec.ini

```diff
@@ -130,6 +94,14 @@ callData: #abiCallData("encodeTransactionData", (
             #address(REFUND_RECEIVER),
             #uint256(NONCE) ))
 callValue: 0
+wordStack: .WordStack => _
+localMem: .IMap => _
+pc: 0 => _
+gas: #gas(0,0) => _
+memoryUsed: 0 => _
```

Comment:
this will be the case even if this function is called inside this contract?
i mean, when execTransaction calls encodeTransactionData, it uses the external call mechanism by spawning a new VM, or just jump to the function locally?
i guess `public` and `external` functions are different...?

Reply:
https://ethereum.stackexchange.com/questions/19380/external-vs-public-best-practices

Reply:
so, this spec cannot be used when proving execTransaction, right?

Reply:
You're correct

Reply:
hmm, then this is not part of our verification scope..   we are interested in this function only in the context of execTransaction.